### PR TITLE
Improved binary packaging using CPack for Windows and macOS

### PR DESCRIPTION
**Pull request by TcT2k**
* Added CPack support for easy packaging in various package formats across various platforms
* Build macOS version as ready to deploy application bundle
* Build macOS in travis-ci
* Build ZIP and MSI installer in appveyor
* Fix about dialog on macOS

See: #264

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,10 @@
 ## Download Binaries
 
 ### Windows
-* wxWidgets 3.0 32-bit [Download](https://ci.appveyor.com/api/projects/jhasse/wxformbuilder-461d5/artifacts/wxFormBuilder_win32.zip?branch=master) (last successful build)
+* wxWidgets 3.1.0 32-bit:
+  [ZIP](https://ci.appveyor.com/api/projects/jhasse/wxformbuilder-461d5/artifacts/wxFormBuilder_win32.zip?branch=master)
```
Comment (jhasse) [inline README.md]: There's a new wxFormBuilder project on appveyor now: https://ci.appveyor.com/project/wxFormBuilder/wxFormBuilder/history
:)

Comment (TcT2k) [inline README.md]: I'll update that URL, it's good to have an easier URL.

**Comment by TcT2k** [comment]
Linux build seems to be broken now, I'll fix that

**Comment by TcT2k** [comment]
Linux build now works again.

**Comment by patlkli** [comment]
Since Travis and Appveyor have successfully built it, I will merge this now. (See #264)

**Pull request merged**